lib/utils/bedroomCalculator.test.ts

```diff
@@ -0,0 +1,51 @@
+import { CALCULATE_BEDROOMS } from './bedroomCalculator';
+
+const people = [
+  [
+    [5, 'male', 'son'],
+    [5, 'female', 'daughter'],
+  ],
+  [
+    [5, 'male', 'son'],
+    [10, 'female', 'daughter'],
+  ],
+  [
+    [5, 'male', 'son'],
+    [15, 'male', 'son'],
+    [10, 'female', 'daughter'],
+  ],
+  [
+    [5, 'male', 'son'],
+    [15, 'male', 'son'],
+    [10, 'female', 'daughter'],
+    [12, 'female', 'daughter'],
+  ],
+  [
+    [5, 'male', 'son'],
+    [5, 'female', 'daughter'],
+    [5, 'female', 'daughter'],
+    [12, 'female', 'daughter'],
+  ],
+  [
+    [30, 'male', null],
+    [25, 'female', 'partner'],
+  ],
+];
+
+describe.only('BedroomCalculator', () => {
+  test('[[5, `male`], [5, `male`]] should be awarded 1 bedroom', () => {
+    expect(CALCULATE_BEDROOMS(people[0])).toBe(1);
```

Comment:
I think you should inline those `people[n]` references. Using numbered indexes like that just makes the code harder to follow.